django/contrib/admin/templates/admin/app_list.html

```diff
@@ -0,0 +1,36 @@
+{% load i18n %}
+
+{% if app_list %}
+    {% for app in app_list %}
+        <div class="app-{{ app.app_label }} module{% if app.app_url in request.path %} current-app{% endif %}">
+        <table>
+        <caption>
+            <a href="{{ app.app_url }}" class="section" title="{% blocktrans with name=app.name %}Models in the {{ name }} application{% endblocktrans %}">{{ app.name }}</a>
+        </caption>
+        {% for model in app.models %}
+            <tr class="model-{{ model.object_name|lower }}{% if model.admin_url in request.path %} current-model{% endif %}">
+            {% if model.admin_url %}
+                <th scope="row"><a href="{{ model.admin_url }}" {% if model.admin_url in request.path %} aria-current="page"{% endif %}>{{ model.name }}</a></th>
+            {% else %}
+                <th scope="row">{{ model.name }}</th>
+            {% endif %}
+
+            {% if model.add_url %}
+                <td><a href="{{ model.add_url }}" class="addlink">{% trans 'Add' %}</a></td>
+            {% else %}
+                <td></td>
+            {% endif %}
+
+            {% if model.admin_url and show_changelinks %}
+                <td><a href="{{ model.admin_url }}" class="changelink">{% trans 'Change' %}</a></td>
+            {% elif show_changelinks %}
+                <td></td>
+            {% endif %}
+            </tr>
+        {% endfor %}
+        </table>
+        </div>
+    {% endfor %}
+{% else %}
+    <p>{% trans "You don't have permission to edit anything." %}</p>
```

Comment:
```suggestion
    <p>{% translate "You don’t have permission to edit anything." %}</p>
```

We should keep the Unicode "Right Single Quotation Mark". It was deliberately added in 42b9a23267f14be39b9b00958e18d5746783208e.

Throughout this file, (multiple lines), the `{% trans %}` block should change to `{% translate %}` to keep commit 35d36d946272bed06a3d7c7cd4e5b71b613e7a4f.

Reply:
Thanks, there were also some other changes missing from this so I've taken the fresh code from master and made my changes over the top of that.